api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2390,6 +2403,27 @@ type ServiceConfig struct {
 	PublicIPSource *PublicIPSource `json:"publicIPSource,omitempty"`
 }
 
+// RoutingConfig allows configuring routing to our pods, and services that sit
+// in front of them.
+type RoutingConfig struct {
+	// Headless determines whether we want to run a headless service for the
+	// cluster.
+	HeadlessService *bool `json:"headlessService,omitempty"`
+
+	// PublicIPSource specifies what source a process should use to get its
+	// public IPs.
+	//
+	// This supports the values `pod` and `service`.
+	PublicIPSource *PublicIPSource `json:"publicIPSource,omitempty"`
+
+	// PodIPIndex tells the pod to use a specific entry from the podIPs list
```

Comment:
Where is the `PodIPIndex`?

Reply:
This is an artifact of an earlier form of the solution.